public/assets/main.css

```diff
@@ -398,3 +398,13 @@ div#game:focus {
     float: left;
   }
 }
+
+.game_process_cell {
+  height: 55px;
+  border: solid 1px rgba(0,0,0,0.2);
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 5px;
+  justify-content: space-between;
+}
```

Comment:
its so much unneeded ugly code if I put it into code. 
whats the reason for no css?

Reply:
In general: Toby prefers code + style in one file and lean main.css. Notable exceptions: It’s way easier to realize with CSS (media queries e.g.) or same styles are used in multiple files.
In this case it’s more or less the same, because you already have def cell_props in place. [… which could be simplified to something like def props(bg, color) for most parts btw.]

Reply:
I understand. Combining code and css helps alot. 
But wouldn't it be best if the framework support some css definitions in the component like vue or angular? But I actually have no idea how complex it is to implement thet in a real good way. (without overriding other css of other components etc.)